tee-worker/ts-tests/bulk_vc.test.ts

```diff
@@ -72,7 +68,7 @@ describeLitentry('multiple accounts test', 10, async (context) => {
     step('test requestVc with multiple accounts', async () => {
         let txs: TransactionSubmit[] = [];
         for (let i = 0; i < substrateSigners.length; i++) {
-            const tx = context.api.tx.vcManagement.requestVc(context.mrEnclave, assertion_type);
+            const tx = context.api.tx.vcManagement.requestVc(context.mrEnclave, assertion_type!);
```

Comment:
Because this [type definition](https://github.com/litentry/litentry-parachain/blob/1673-use-generated-types/tee-worker/ts-tests/common/type-definitions.ts#L100) uses the "?" operator.

Reply:
By its use in `VCRequested` I'm assuming that the `Assertion` type is supposed to encode _one_ assertion with its params, correct? Then shouldn't it rather be sth like

```
type Assertion = {
  A1: string;
} | {
  A2: [string]
} | ...
```
?

Reply:
I agree with your suggestion. Union types are indeed much better than optional types.Fixed it.